core/templates/pages/exploration-editor-page/translation-tab/services/translation-language.service.ts

```diff
@@ -76,9 +80,29 @@ export class TranslationLanguageService {
     );
   }
 
+  // Function returns null when active language accent code is not set.
+  getActiveLanguageAccentCode(): string | null {
+    if (!this.activeLanguageAccentCode) {
+      return null;
+    }
+    return this.activeLanguageAccentCode;
```

Comment:
Couldn't we just do 
`return this.activeLanguageAccentCode`, The if statement here seems redundant, assuming this.activeLanguageAccentCode is null when it isn't set.

Reply:
Yes we can do that, thank you @Lawful2002 for pointing this. Done